change/@ni-nimble-angular-08a8d89d-f416-4aa9-8946-711f3c495d75.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Since the table is marked "experimental" we've been using "patch" even when making minor or breaking changes in order to reduce churn on clients of other components.

Reply:
Fixed